music-library-team2/Model/GenreMenu.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace music_library_team2.Model
+{
+   public class GenreMenu
+    {
+
+        
+        public Genre GenreCategory { get; set; }
```

Comment:
why do you need to define this class?

Reply:
To filter the music by their genre and display every song  by their genre, when click on it